src/systems/a320_systems/src/pneumatic.rs

```diff
@@ -12,6 +12,7 @@ use uom::si::{
 
 use systems::{
     accept_iterable,
+    hydraulic::update_iterator::MaxStepLoop,
```

Comment:
Should we move those loop updaters to shared?

Reply:
Good point, we probably should. I can do that 👍